package.json

```diff
@@ -22,7 +22,8 @@
     "clean": "rm -rf ./node_modules && rm -rf ./packages/*/node_modules",
     "postinstall": "npm run bootstrap",
     "bootstrap": "lerna bootstrap",
-    "initDB": "npm run build && node ./packages/ldes-manager/bin/orm/init.js"
+    "initDB": "npm run build && node ./packages/ldes-manager/bin/orm/init.js",
```

Comment:
Maybe it's an idea to use migrations for the database. See https://sequelize.org/master/manual/migrations.html

Reply:
I am not sure on how to do that since I didn't use the `sequelize-cli`